### ISSUE TITLE
Image links bug?

### ISSUE THREAD

**Issue by jjr03** [haoict/twitter-no-ads#10]
Why does these happen? Cover images of links won't show up. 
![F4EE9166-80D4-4056-8F56-9C5E874D5DE4](https://user-images.githubusercontent.com/67056256/84897266-e682a400-b0d7-11ea-9a76-1f7474598217.png)

**Comment by haoict** [comment]
Yeah this is known bug, you can temporary disable "Skip Analytic URL" in preferences to show the preview again
